src/robot/testdoc.py

```diff
@@ -53,6 +53,7 @@
   -s --suite name *      Include suites by name.
   -i --include tag *     Include tests by tags.
   -e --exclude tag *     Exclude tests by tags.
+  -S --skip tag *        Skip tests by tags.
```

Comment:
That depends on the final implementation of --skip.

I have implemented the skipping as much as I can but I feel Pekkaklarck I cannot take it further
as my understanding of the robot source just is not good enough to take this further. I do hope
that what I have provide will allow you (or someone else in the robot development team to take
skipping to the wider Robot Framework communing in a comming release).

Regards,
Doug